windows.md

```diff
@@ -193,8 +193,28 @@ With those compatibility things out of the way, you're ready to start the system
 
     1. Find `shell: '',` and replace it with `shell: 'C:\\Program Files\\Git\\bin\\bash.exe',`
     2. Find `env: {},` and replace it with `env: { TERM: 'cygwin' },`
-
-    Then save the file and close and restart Hyper.
+    3. Save the file and close and restart Hyper.
+    4. To verify that the setup was successful, you can copy and paste the following command into Hyper and hit return:
+    
+        ```bash
+        echo $SHELL; echo $TERM
```

Comment:
let's avoid semicolons - we don't use these in our commands

Reply:
Changed the command to a single echo and added a prefix
[129e30b](https://github.com/upleveled/system-setup/pull/32/commits/129e30b1b6944889da8ae65f20da318282e99710)